paraview/singlePlugin/CMakeLists.txt

```diff
@@ -17,6 +17,8 @@ paraview_add_plugin(TopologyToolKit
   )
 
 if(NOT "${TTK_INSTALL_PLUGIN_DIR}" STREQUAL "")
+  ttk_set_paraview_rpath(TopologyToolKit)
```

Comment:
Isn't this the `${TTK_PLUGIN_SUBDIR}` ?

Reply:
Actually it's not. That's the CMake target for `TopologyToolKit.so`, which happens to be `TopologyToolKit`.

`${TTK_PLUGIN_SUBDIR}` is supposedly the subdirectory, which is also named `TopologyToolKit`. This dir contains the libttk*.so files that have the actual computation code.